protocols/grpc/hyperstart.proto

```diff
@@ -108,20 +111,31 @@ message CreateSandboxRequest {
 message DestroySandboxRequest {
 }
 
+enum IpFamily {
+	v4 = 0;
+	v6 = 1;
+}
+
 message IpAddress {
-	string address = 1;
-	string mask = 2;
+	IpFamily family = 1;
+	string address = 2;
+	string mask = 3;
 }
 
-message UpdateInterfaceRequest {
-	uint64 type	=1;
```

Comment:
I checked the previous runV code for this field. And found it defined the type operations instead of interface. The operations include:

```
+type InfUpdateType uint64
 +
 +const (
 +	AddInf InfUpdateType = 1 << iota
 +	DelInf
 +	AddIP
 +	DelIP
 +	SetName
 +	SetMtu
 +)
```

This API is contributed by Wei from huawei in this commit: hyperhq/runv@84e1f86f40d5dd2d3ced128d92008bf06f1ca913  of PR hyperhq/runv#597 .

In hyperhq/runv#597 , Wei contributed a `interface` command, with which runV could work with external CNI script directly.

Reply:
So we still need `AddIP`, `DelIP`, `SetName` and `SetMtu`? If so, we'd better define them clearly.

Reply:
What I mean here is -- the new API proposed in this API is in total different flavor:

- the new API is declarative, which include all the interface description in the request;
- the original one is an incremental API, which may only ships the changed value. For example, if we do `SetMtu`, we may know nothing about the IP addresses.

Reply:
@gnawux Good point, thanks. Does the `UpdateInterfacesRequest` look better ?